src/ralph/admin/mixins.py

```diff
@@ -85,11 +85,24 @@ def changelist_view(self, request, extra_context=None):
         for view in self.list_views:
             views.append(view)
         extra_context['list_views'] = views
+        if self.get_actions(request) or self.list_filter:
+            extra_context['has_filters'] = True
+
+        extra_context['bulk_edit'] = request.GET.get(BULK_EDIT_VAR, False)
+        if extra_context['bulk_edit']:
+            extra_context['has_filters'] = False
```

Comment:
template var ``has_filters`` is strong attached to visibility sidebar - maybe rename this var to ``has_sidebar``? @szok what do you think?